ui/modules/mon-espace/effectifs/Televersements.jsx

```diff
@@ -29,10 +29,10 @@ import { effectifsStateAtom } from "./engine/atoms";
 import Ribbons from "../../../components/Ribbons/Ribbons";
 
 const Televersements = () => {
-  useFetchUploads();
   const { documents, uploads, onDocumentsChanged } = useDocuments();
   const [step, setStep] = useState("landing");
   const organisme = useRecoilValue(organismeAtom);
+  useFetchUploads(organisme._id);
```

Comment:
attention, j'ai remarqué que parfois organismeAtom n'est pas encore initialisé, du coup organisme est null. Peut etre pas le cas la, mais peut etre a anticiper

Reply:
J'ai modifié le hook pour ne rien faire si pas organismeId n'est pas défini. Car apparemment il ne faut pas mettre de condition autour d'un appel à un hook

En tout cas quand on navigue un peu sur l'UI en dev, j'ai de belles erreurs dans les logs, mais c'est plus général j'ai l'impression. :roll_eyes: 

![image](https://user-images.githubusercontent.com/8938024/218126957-d41cdcae-1af4-434d-b31d-551925fc114e.png)

Reply:
Oui, je vois cette erreur assez regulierement.

https://stackoverflow.com/questions/65506656/recoil-duplicate-atom-key-in-nextjs